backend/src/packages/sites/site.repository.ts

```diff
@@ -17,6 +17,15 @@ class SiteRepository
     return sites.map((site) => SiteEntity.initialize(site));
   }
 
+  public async findByProjectId(projectId: number): Promise<SiteEntity[]> {
+    const sites = await this.siteModel
+      .query()
+      .where('project_id', projectId)
```

Comment:
Within the snakeCaseMapper we may define keys in non-snake_case(snakeCase) for not a raw queries